content/en-us/reference/engine/classes/PluginToolbarButton.yaml

```diff
@@ -67,6 +67,10 @@ properties:
       `Class.PluginToolbarButton.Click|Click` event. Buttons are enabled by
       default.
 
+      When re-enabling this property, the plugin button's state won't be remembered from the previous state in which the user left the button in.
+      Instead, it will default to the last state set by `Class.PluginToolbarButton:SetActive()|SetActive`
+      or to the inactive state if `Class.PluginToolbarButton:SetActive()|SetActive` was never used.
```

Comment:
```suggestion
      When re-enabling this property, the plugin button's state won't be
      remembered from the previous state in which the user left the button
      in. Instead, it will default to the last state set by
      `Class.PluginToolbarButton:SetActive()|SetActive()` or to the inactive
      state if `Class.PluginToolbarButton:SetActive()|SetActive()` was
      never used.
```